software/django-crm.yml

```diff
@@ -0,0 +1,10 @@
+name: Django-CRM
+website_url: https://github.com/DjangoCRM/django-crm
+source_code_url: https://github.com/DjangoCRM/django-crm
+description: Analytical CRM with tasks management, email marketing and many more. Django CRM is built for individual use, businesses of any size or freelancers and is designed to provide easy customization and quick development.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Python
+tags:
+  - Groupware
```

Comment:
I don't think `Groupware` is quite correct here but we don't have a CRM tag/category currently (3 relevant software items are required to add a new tag) . If you can find 2 othe CRM solutions in the list I'd add a category. Otherwise we should probably move it to `Miscellaneous` but I'd rather avoid this...